api/swagger.yaml

```diff
@@ -2195,9 +2195,6 @@ definitions:
         description: |
           Number of containers using this image. Includes both stopped and running
           containers.
-
-          This size is not calculated by default, and depends on which API endpoint
-          is used. `-1` indicates that the value has not been set / calculated.
```

Comment:
> `-1` indicates that the value has not been set / calculated.

We could keep this; even though we no longer return by default, but to explain it could be